packages/parser/tests/unit/schema/utils.ts

```diff
@@ -0,0 +1,11 @@
+import * as fs from 'fs';
+
+export const loadExampleEvent = (fileName: string): unknown => {
+  try {
+    const event = fs.readFileSync(`tests/events/${fileName}`, 'utf8');
+
+    return JSON.parse(event);
+  } catch (err) {
+    console.error(err);
```

Comment:
I'd consider throwing and removing the log here, at least the test will fail with an error that tells us there was an issue reading the file rather than emitting a log somewhere in between the lines & then fail with an Zod error later

Reply:
Yeah, didn't check.